xlators/cluster/afr/src/afr.c

```diff
@@ -1326,7 +1326,7 @@ struct volume_options options[] = {
     {.key = {"use-anonymous-inode"},
      .type = GF_OPTION_TYPE_BOOL,
      .default_value = "no",
-     .op_version = {GD_OP_VERSION_8_0},
+     .op_version = {GD_OP_VERSION_9_0},
```

Comment:
Just to understand, this is how it should have been even in the original commit  2d58ec85816d3f03cfde23eea8b8ba03ce50a3a6  right?

Reply:
Yeah, it was a miss, but this doesn't have any repercussions.